framework/platform/CMakeLists.txt

```
@@ -1 +1,4 @@
-add_subdirectory(bpl)
\ No newline at end of file
+add_subdirectory(bpl)
+if (ENABLE_AMBIORIX)
+    add_subdirectory(HLAPI)
```

Consistency is important.
Better name this sub directory with lower case letters
```suggestion
    add_subdirectory(hlapi)
```

---

Applied! Resolving.